simulator/func_sim/rf/rf.h

```diff
@@ -109,10 +132,12 @@ class RF
         Register reg_num  = instr.get_dst_num();
         bool writes_dst = instr.get_writes_dst();
         auto accumulating_instr = instr.is_accumulating_instr();
+        auto loadlr = instr.is_loadlr();
+        auto offset = instr.get_v_imm();
```

Comment:
It seems you ignored my remark that immediate is not the only source of alignment, right?

As far as I understand that code is valid as well, but 

```asm
some_label:
; some data

la $at, some_label
addi $at, $at, 2 ; generating an address with (addr % 4) == 2
lwl $at, 0($at) ; reading from unaligned address
```

Reply:
Oh, sorry, I didn't see your question when I started the review